core/backend/src/TxnManager.ts

```diff
@@ -292,6 +303,7 @@ export class TxnManager {
 
   /** @internal */
   protected _onCommitted() {
+    this.touchWatchFile();
```

Comment:
i realize this PR is already committed, but I think you should only touch the watch file when the briefcase is closed. This will avoid the problem with undo/redo and is all we need. Touching the file on the undo/redo event won't work because we never look for the Txns going backwards.